### PR TITLE
test: enter notes on ivy-related material test errors

### PR DESCRIPTION
**Pull request by andrewseguin**
Updates the latest findings of the material2 test errors when running with ivy

### CODE REVIEW

**Review by jelbourn — APPROVED** [review]
LGTM

File: tools/material-ci/angular_material_test_blacklist.js [hunk 1]
```diff
@@ -35,87 +35,87 @@ window.testBlacklist = {
   },
   "Portals CdkPortalOutlet should be considered attached when attaching using `attach`": {
     "error": "TypeError: Cannot read property 'hasAttached' of undefined",
-    "notes": "Unknown"
+    "notes": "FW-1019: Need to perform static query"
```
Comment (kara) [inline tools/material-ci/angular_material_test_blacklist.js]: Can we have the summary here match the name of the Jira ticket? So it's easily searchable?

**Review by kara — APPROVED** [review]
LGTM, pending one nit

File: tools/material-ci/angular_material_test_blacklist.js (hunk 1 above)
Comment (andrewseguin) [reply tools/material-ci/angular_material_test_blacklist.js]: Sure thing - I updated the notes to reflect the Jira ticket's title

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
merge-assistance: one core approval should be sufficient for this file (the dev-infra group was requested before the CODEOWNERS change, https://github.com/angular/angular/pull/28531, went in)

**Pull request closed without merging**